actors/common/src/main/java/org/sunbird/learner/util/UserFlagUtil.java

```diff
@@ -10,19 +10,22 @@
   /**
    * This method return int value of the boolean flag
    * @param userFlagType
-   * @param isFlagEnabled
+   * @param flagEnabled
    * @return
    */
-  public static int getFlagValue(String userFlagType, boolean isFlagEnabled) {
+  public static int getFlagValue(String userFlagType, boolean flagEnabled) {
     int decimalValue = 0;
+    //if phone is verified, value is set to true
     if(userFlagType.equals(UserFlagEnum.PHONE_VERIFIED.getUserFlagType()) &&
-            isFlagEnabled== UserFlagEnum.PHONE_VERIFIED.isFlagEnabled()) {
+            flagEnabled == true) {
```

Comment:
flagEnabled == true  to flagEnabled  only.

Reply:
Done.